src/ui/buttons.html

```diff
@@ -21,11 +21,16 @@
     <script src="js/extern/jquery-1.9.0.min.js"></script>
     <script>
         $(document).ready(function() {
-          $.getScript('js/Loader.js', function() {
-            Loader.loadScripts([ 'js/Buttons.js' ], function() {
-                Login.showLoginHeader(Buttons);
+            $.ajax({
+                url: 'js/Loader.js',
+                dataType: 'script',
+                cache: true,
+                success: function() {
+                    Loader.loadScripts([ 'js/Buttons.js' ], function() {
+                        Login.showLoginHeader(Buttons);
+                    });
+                }
```

Comment:
Ah, interesting. I was trying to work out what the correct indentation was, since the HTML is indented with 4 spaces, but the JS was indented with 2 spaces.

Any thoughts on this? I thought it was odd to have mixed indentation.